### ISSUE TITLE
Scourge Arrow Thorn Arrows Missing 0.2 Multiplier

### ISSUE THREAD

**Issue by qikink** [Openarl/PathOfBuilding#1396]
When calculating the DPS of Scourge Arrow's Thorn Arrows part, the 750% More from 5 stages is taken into account, however the calc is missing the multiplication by 0.2 to account for charging to 5 stages that the "Release at 5 stages" skill part has in the "Skill DPS" layer of the calc.

**Comment by LucasAzais** [comment]
Yes I was looking into something else and noticed it as well. It multiplies the average damage by the attack speed only.

The fix is in the skill data (act_dex.lua), adding skill part 3 to the DPS multiplier mod.

I'll make a PR for it.
